pkg/version/version.go

```diff
@@ -47,7 +47,7 @@ var (
 	TimescaleVersionRangeString = struct {
 		Safe, Warn string
 	}{
-		Safe: ">=1.7.3 <2.0.0",
+		Safe: ">=1.7.3 <=2.0.0",
```

Comment:
@cevian What is support matrix for timescaleDB? Is it latest (i.e. 2.0.0) -1 (i.e. 1.7.4) or -2 (i.e. 1.7.3) ?